lib/active_record_shards.rb

```diff
@@ -24,10 +25,10 @@ def self.rails_env
 ActiveRecord::Base.extend(ActiveRecordShards::DefaultSlavePatches)
 ActiveRecord::Relation.include(ActiveRecordShards::DefaultSlavePatches::ActiveRelationPatches)
 ActiveRecord::Associations::CollectionProxy.include(ActiveRecordShards::AssociationCollectionConnectionSelection)
+ActiveRecord::Associations::Builder::HasAndBelongsToMany.include(ActiveRecordShards::DefaultSlavePatches::Rails41HasAndBelongsToManyBuilderExtension)
+ActiveRecord::SchemaDumper.prepend(ActiveRecordShards::SchemaDumperExtension)
```

Comment:
These lines are moved from lib/active_record_shards/patches-4-2.rb and lib/active_record_shards/patches-5-0.rb